### ISSUE TITLE
possible to create a heading compass based on the windrose card?

### ISSUE THREAD

**Issue by vcanabal** [aukedejong/lovelace-windrose-card#80]
Use Case: Boat

I love the new functionality of being able to dynamically set the compass direction based on a sensor.  Is it possible to create a dedicated compass card based on this?  Converting the wind direction arrow to a outward facing arrow pinned at the 12 o'clock position, a more detailed compass (more ticks and cardinal direction labels), and have the compass rotate with a dynamic offest based on a heading sensor.

**Comment by aukedejong** [comment]
HI,
I would like to create a compass card with the features you describe. The problem is time.
Maybe later this year, but I can't promise anything.

I want to complete some feature requests for the windrose-card first.

**Comment by vcanabal** [comment]
Makes sense and thanks so much!  The windrose card is incredible and I really like being able to dynamically set the offset!  Made me think this would be a great boat compass!

**Comment by platini76** [comment]
any news about this?
